docs/contributing-to-airbyte/developing-on-docker.md

```diff
@@ -0,0 +1,46 @@
+# Developing on docker
+
+## Version
+
+This doc is relevant starting from the version 0.30.28-alpha. Prior to that, it was another design.
```

Comment:
Can remove this section since nobody cares about the prior design.

Reply:
Ok, That's surprising I would have expect something like `Since` in the javadoc,